buffer/src/main/java/io/netty/buffer/PoolChunk.java

```diff
@@ -610,7 +608,7 @@ void initBufWithSubpage(PooledByteBuf<T> buf, ByteBuffer nioBuffer, long handle,
         int bitmapIdx = bitmapIdx(handle);
 
         PoolSubpage<T> s = subpages[runOffset];
-        assert s.doNotDestroy;
+        assert s.doNotDestroy();
```

Comment:
Imho it's very confusing that we have a method and also a field that is called `doNotDestroy`.

Reply:
em... What about `isDoNotDestroy()`?